client/packages/common/src/styles/theme.ts

```diff
@@ -193,6 +194,7 @@ export const themeOptions = {
       menu: '#f2f2f5',
       toolbar: '#fafafc',
       white: '#fff',
+      success: 'rgb(237, 247, 237)',
```

Comment:
This the pale green used in the Alert backgrounds. I couldn't find anywhere that it's defined, it's just a Material defaul -- the general `success.main` colour defined is just the darker icon colour. So I've just copied it from the CSS here for re-use later. If anyone knows how to access the default Material colour definitions directly, that'd be cool.